src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProject.cs

```diff
@@ -156,6 +157,12 @@ internal sealed partial class ProjectSystemProject
         Language = language;
         _displayName = displayName;
 
+        var provider = _projectSystemProjectFactory.Workspace.Services.GetRequiredService<IAnalyzerAssemblyLoaderProvider>();
+        // Shadow copy analyzer files coming from packages to avoid locking the files in NuGet cache.
+        // NOTE: The provider will always return the same singleton analyzer loader instance, which is important to
+        // ensure that shadow copied analyzer dependencies are correctly loaded.
+        _analyzerAssemblyLoader = provider.GetLoader(shadowCopy: true);
```

Comment:
So do we pass false for shadow copy anywhere? Because I'm not really sure why that flag exists. It seems the shadow copy or not flag is really a host question more or less...or is the flag a "you might want to shadow copy" hint and saying no definitely opts out?

Reply:
We potentially do here - https://github.com/dotnet/roslyn/blob/f34037a40f100c98bf97db83ddb9578f1c9547f7/src/Workspaces/Core/Portable/Serialization/SerializerService_Reference.cs#L72

I couldn't track down though where that could be coming from.  For now will leave as-is.

Reply:
Want to file a bug to track?